packages/governance/src/contractGovernor.js

```diff
@@ -173,7 +173,7 @@ const start = async zcf => {
     return poserFacet;
   };
   await getUpdatedPoserFacet();
-  assert(poserFacet, X`question poser facet must be initialized`);
+  assert(poserFacet, 'question poser facet must be initialized');
```

Comment:
would your lint rule require that details templates not be used when the expression is a string literal?

Reply:
> I wouldn't require that, since sometimes the message is still being massaged, and is likely to gain or lose details.

Good! That's a nice rationale for not including the "it must have sub-expressions" that I mentioned I'm uncomfortable with below.

Resolving.